test/integration/freetext_editor_spec.mjs

```diff
@@ -27,7 +27,6 @@ import {
   getFirstSerialized,
   getRect,
   getSerialized,
-  hover,
```

Comment:
I think this helper function isn't used anywhere else anymore after it's removed here. If so, please also remove the implementation from `test_utils.mjs`.